src/server/rpc/procedures/water-watch/water-watch.js

```diff
@@ -16,7 +16,15 @@ function encodeQueryData(options) {
     return ret.join('&');
 }
 
-
+/**
+ * Get the water data for sites within a bounding box.
+ * For help interpreting this data, see https://help.waterdata.usgs.gov/tutorials/surface-water-data/how-do-i-interpret-gage-height-and-streamflow-values
+ * @param {Latitude} minLatitude Minimum latitude of bounding box
+ * @param {Latitude} maxLatitude Maximum latitude of bounding box
+ * @param {Longitude} minLongitude Minimum longitude of bounding box
+ * @param {Longitude} maxLongitude Maximum longitude of bounding box
+ * @returns {Promise<Object>} Messages sent of data
```

Comment:
The return value for this should be `Promise<String>` since `_sendMsgs` returns a short description including the number of messages and the message type.

Reply:
This is a great example of what we were talking about yesterday.

Reply:
@gsteinLTU are you talking about clickable links in help docs? if so it could be useful for charts services too

Reply:
@gsteinLTU Unfortunately, this PR is made from your fork (which I don't have write access to).

Reply:
That's fine then, I'll see if I can do it from here since the weather has me stuck at home.

@hamidzr No, we were talking about where we have to go back and forth with changes like this that are just a simple find and replace. It's just making extra work for both of us, since he already knows the exact change that will be made (and has to spend more time commenting than he would doing it), there won't be any need for choice or discussion (it's a demand, not a request), and he'll have to review it again to make sure it's what he wanted. In this case it doesn't matter, but going forward we should avoid that.

Reply:
I take that back, you do have access as part of the PR.